mods/ra/maps/soviet-13a/map.yaml

```
@@ -0,0 +1,939 @@
+MapFormat: 12
+
+RequiresMod: ra
+
+Title: 13a: Capture the Chronosphere
+
+Author: Westwood Studios
+
+Tileset: SNOW
+
+MapSize: 128,128
+
+Bounds: 18,17,96,90
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: england
+	PlayerReference@Greece:
+		Name: Greece
```

I won't pretend to fully understand why, but giving Greece a bot seems to fix the idle helicopters that @PunkPun  mentioned. Without it, Longbows can skip the usual ReturnToBase after using up their ammo pool, and they'll just hover.

Other than that, I ran through it twice today without any issues coming from the script.
```suggestion
		Name: Greece
		Bot: campaign
```

---

Do we do this in any of the other missions though?

---

Soviet-01, Soviet-02a, Soviet-02b, Soviet-03, Soviet-04a, Soviet-04b, Soviet-05 (with a tweaked skirmish bot for the island), Soviet-06a, Soviet-06b, Soviet-07, Soviet-08a, Soviet-08b, Soviet-09, and Soviet-10 all seem to do this. Most of the Allies missions have campaign bots too.

---

Interesting, did you test for any potential unwanted side effects?

---

I did run through the map a third time without any apparent issues from the campaign bot, after I had put together a small test map to confirm the different behavior. While I was at it, I also used the test map to see if the clogged factory exit was related, and it didn't seem to be.